build.cake

```diff
@@ -1,135 +1,45 @@
+static string Target; Target = Argument("target", "Default");
+static string Configuration; Configuration = Argument("configuration", "Release");
```

Comment:
What happens if you try `static readonly string Configuration = Argument("configuration", "Release");`?

Reply:
That's fair. Some things can't be readonly, of course, but I got carried away.

Reply:
For those things, just a shortening of the syntax then, like:

```suggestion
static string Configuration = Argument("configuration", "Release");
```